components/StatusPage/StatusPage.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: "StatusPage"
+description: "A widget that displays the operational status of a service."
+author: "@trishaprile"
+version: "1.0.0"
+---
+
+import { useEffect, useState } from 'react';
+
+export const StatusPage = ({ title, url}) => {
+  const [status, setStatus] = useState(null);
+  const [indicator, setIndicator] = useState(null);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    if (!url) return;
+
+    const fetchStatus = async () => {
+      try {
+        const res = await fetch(`${url.replace(/\/$/, '')}/api/v2/status.json`);
+        if (!res.ok) throw new Error(`Failed to fetch status: ${res.status}`);
+        const data = await res.json();
+        setStatus(data.status.description);
+        setIndicator(data.status.indicator);
+      } catch (err) {
+        setError('Unable to fetch status');
+      }
+    };
+
+    fetchStatus();
+  }, [url]);
```

Comment:
This should probably check if the component has been unmounted, like when switching pages.

```suggestion
  useEffect(() => {
    if (!url) return;
    let isMounted = true;

    const fetchStatus = async () => {
      try {
        const res = await fetch(`${url.replace(/\/$/, '')}/api/v2/status.json`);
        if (!res.ok) throw new Error(`Failed to fetch status: ${res.status}`);
        const data = await res.json();
        if (!isMounted) return;
        
        setStatus(data.status.description);
        setIndicator(data.status.indicator);
      } catch (err) {
        if (!isMounted) return;
        
        setError('Unable to fetch status');
      }
    };

    fetchStatus();
    
    return () => {
      isMounted = false;
    };
  }, [url]);
```